
export async function scan() {
    try {
        console.log("In scan")
    } catch (error) {
        console.error("Error occurred while reading the directory!", error);
    }
}
